src/less/skeleton/skeleton.less

```diff
@@ -0,0 +1,227 @@
+@import "../variables/variables.less";
+@import "../mixins/public/skeleton-mixins.less";
+
+@skeleton-text-border-radius: 3px;
+@skeleton-text-height-small: 16px;
+@skeleton-text-height-large: 24px;
+
+.skeleton-avatar {
+    .skeleton-base(50%, var(--skeleton-height), var(--skeleton-height));
+}
+
+.skeleton-button {
+    .skeleton-base(20px, 40px);
+}
+
+.skeleton-button--small {
+    .skeleton-base(16px, 32px);
+}
+
+.skeleton-button--large {
+    .skeleton-base(24px, 48px);
+}
+
+.skeleton-text,
+.skeleton-text--small {
+    .skeleton-base(@skeleton-text-border-radius, @skeleton-text-height-small);
+}
+
+.skeleton-text::after,
+.skeleton-text--small::after {
+    height: @skeleton-text-height-small;
+    margin-top: calc(@skeleton-text-height-small + var(--spacing-100));
+}
+
+.skeleton-text--large {
+    .skeleton-base(@skeleton-text-border-radius, @skeleton-text-height-large);
+}
+
+.skeleton-text--large::after {
+    height: @skeleton-text-height-large;
+    margin-top: calc(@skeleton-text-height-large + var(--spacing-100));
+}
+
+.skeleton-text--multiline {
+    margin-bottom: var(--spacing-300);
+    position: relative;
+    width: calc(100% - var(--spacing-300));
+}
+
+.skeleton-text--multiline::after {
+    background: inherit;
+    content: "";
+    position: absolute;
+    width: calc(100% - var(--spacing-700));
+}
+
+.skeleton-text--large.skeleton-text--multiline {
+    margin-bottom: var(--spacing-400);
+}
+
+.skeleton-form {
+    .skeleton-base(8px, 48px);
+}
+
+// TODO: Use aspect-ratio : 1 / 1 to set as square once browser support for safari > 15
+.skeleton-image {
+    .skeleton-base(8px, 100%);
+    container-name: skeleton-image;
+    container-type: inline-size;
+    max-width: initial;
+}
+
+.skeleton-image::after {
+    bottom: 0;
+    height: 100%;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    width: 100%;
+}
+
+span.skeleton-avatar,
+span.skeleton-button,
+span.skeleton-text,
+span.skeleton-image,
+span.skeleton-form {
+    display: inline-block;
+}
+
+span.skeleton-avatar + span,
+span.skeleton-button + span,
+span.skeleton-text + span,
+span.skeleton-image + span,
+span.skeleton-form + span {
+    margin-inline-start: var(--spacing-100);
+}
+
+div.skeleton-avatar,
+div.skeleton-button,
+div.skeleton-text,
+div.skeleton-image,
+div.skeleton-form {
+    margin-block-start: var(--spacing-150);
+}
+
+.skeleton-tile {
```

Comment:
The more I look at this, I agree that this might need to be removed. But I checked Figma again and there is a certain spacing requirement on how widths need to be set for text in tiles for consistency. If we remove this from skin and show an example, there might be that inconsistent spacing as teams won't refer figma for specs and they might just use our example as reference. @ianmcburnie any thoughts?